index.html

```diff
@@ -242,6 +242,31 @@ <h2 class="big-text">About Myself</h2>
           <hr />
         </div>
       </section>
+      <!--section form of the portfolio cards starts-->
+    <section class="sectionf">
+      <div class="SectionfText">
+        <h2 class="frm">Contact me</h2>
+        <p class="center"> If you have application you are interested in developing,
+          a feature that you need built or a project that needs 
+          coding. I'd love to help with it</p>
+          <form action="https://formspree.io/f/meqnzbev" target="_blank" method="post">
+            <p><input id="username" type="text" class="username" name="username" maxlength="30"   placeholder="Yeremias" required>
+            </p>
+            <p>
+                <input id="email" class="email" type="email" name="email"  placeholder="natajayanj14@gmail.com" required>
+            </p> 
+            <p> 
+              <textarea name="message" placeholder="write your message here" class="textarea" maxlength="500"  id="message" cols="30" rows="10" required></textarea>
+            </p>
+            <p>
+              <input type="submit" class="form-btn" value="Get It Touch" >
+            </p>
+          </form>
+         <p>
+          <img class="imgf" src="img/Shape.svg" alt="gym">
+         </p>
+       </div>
+  </section>
```

Comment:
- Kindly try to stick as much as possible to the Figma design, which includes:
   - [ ] Adjusting the spacing around each element in your page to match the Figma design.
   - [ ] Adjusting the background image positioning to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/159687876-3f651dbf-6632-46f7-b68b-7ad95893dab2.png) | ![image](https://user-images.githubusercontent.com/62290545/159688205-c639089a-4335-4fa6-9fb6-2f4ac8ecbbca.png) |